go.mod

```diff
@@ -5,6 +5,7 @@ go 1.22
 require (
 	github.com/cockroachdb/pebble v1.1.2
```

Comment:
Nit: pebble is a heavy dependency. If not used by this module yet, consider deferring adding it to reduce module bloat for consumers (unless used elsewhere in repo).